packages/react-components/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@fluentui/react-components",
+  "version": "9.0.0-alpha.0",
+  "description": "Suite package for converged React components",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
```

Comment:
Since `@fluentui/set-version` is a v8 package, I wonder if it would be best to just remove that dependency and related config for now (from this new package, all existing converged packages, and the template). Then in the future if we decide we need similar functionality we can add it back.

(The purpose of set-version is to provide a `__packages__` global that contains all the versions of packages loaded on the page to help with debugging, especially to figure out if there are multiple versions of a package being loaded.)